_data/projects/project-catalog.yml

```
@@ -0,0 +1,8 @@
+name: Project Catalog
+desc: Project-catalog is a web app created using next js. A place where university students can upload thier project info , images , markdown file and other users may search for projects as a reference.
+site: https://nextjs-project-manager-phi.vercel.app
+tags:
+  nextjs , mantine-ui , google analytics 4, 
```

This is how to do an array in YAML

```suggestion
   - nextjs
   - mantine-ui
   - google-analytics
```